calliope/test/common/test_model/group_constraints.yaml

```diff
@@ -0,0 +1,55 @@
+model:
```

Comment:
This seems like an unnecessary file addition... Why not just set 'exists' to False as an override dict in the specific test, rather than have a whole new YAML file?

Reply:
Agreed, if it was only for this one override, the file wouldn't be necessary.

But I also added that file in #194 with more overrides. The idea being, that there is one test file + one test class for the general mechanism of group constraints: application to several locations, application to several technologies, switchability, etc. Other files and test classes would then test specifically the constraints, not the group constraint mechanism. Does that make sense, or would you rather mix the general tests into the other files?

Reply:
Aha, OK. It's a bit messy at the moment, since the testing of location/technology subsets happens in the individual constraint model overrides anyway (which is valid, to check that the constraint is working correctly). Let's pass all these through, then once we have all the group constraints merged together, we can look at consolidating if it looks a bit much.

Reply:
👍